ocs_ci/ocs/resources/ocs.py

```diff
@@ -15,6 +15,9 @@ class OCS(object):
     """
     Base OCSClass
     """
+    # This _delete flag is set to True if the delete method was called on
+    # object of this class and was successfull.
+    _deleted = False
```

Comment:
s/_deleted/deleted or is_deleted, since we are planning to use it outside class.

Reply:
Done by property